src/moi_wrapper.jl

```diff
@@ -50,6 +50,19 @@ function diff_optimizer(
         with_bridge_type,
         with_cache_type,
     )
+    if !isnothing(with_bridge_type)
+        # removed because of the `ZerosBridge` issue:
+        # https://github.com/jump-dev/MathOptInterface.jl/issues/2861
+        # - zeros bridge does not support duals because it cumbersome
```

Comment:
cumbersome ? Shouldn't we simply say that ZerosBridge does not support dual and we know with DiffOpt that we will need the duals so we can't use that bridge ?

Reply:
This bridge is only useful in situations where we don't need the dual so in case we are sure in advance that we need the dual it's clear we can remove it